draft-packages/modal/KaizenDraft/Modal/Primitives/ModalAccessibleLabel.tsx

```diff
@@ -11,7 +12,11 @@ type ModalAccessibleLabel = React.FunctionComponent<ModalAccessibleLabelProps>
 const ModalAccessibleLabel: ModalAccessibleLabel = ({ children }) => (
   <ModalAccessibleContext.Consumer>
     {({ labelledByID }) => (
-      <div id={labelledByID} tabIndex={-1} className={styles.label}>
+      <div
+        id={labelledByID}
+        tabIndex={-1}
+        className={classnames(styles.modalLabel, {})}
```

Comment:
Why the empty object?

Reply:
There will be something in that object in the next PR that is Confirmation Modal related so when refactoring I just removed the insides so that this PR is just Input Edit stuff so Yes shouldn't be there but I guess ignore for now :)